README.md

```diff
@@ -42,9 +42,11 @@ Where a field definition object takes the following form:
   placeholder: String data (optional)
   metered: boolean (optional),
   optional: boolean (optional),
+  options: array of strings for each option when "type" is choiceGroup, checkboxGroup, or a React component that takes "options" (like a ReactSelect component),
```

Comment:
yeah it's the various options that you can select. If it's a radio group, you can only pick one, if it's checkboxes, then you can select multiple, and if it's something like a pull down menu, think `<select>` options.

So you can specify something like:
```
  type: "checkboxGroup",
  options: ["toddler", "preteen", "teen", "young adult", "adult"],
  ...
```
and then the UI would show something like

- [ ] toddler
- [ ] toddler
- [ ] toddler